jdbc-handler/src/main/java/org/apache/hive/storage/jdbc/JdbcRecordWriter.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hive.storage.jdbc;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.SQLException;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.hadoop.hive.ql.exec.FileSinkOperator.RecordWriter;
+import org.apache.hadoop.io.Writable;
+import org.apache.hadoop.mapreduce.lib.db.DBOutputFormat.DBRecordWriter;
+import org.apache.hadoop.util.StringUtils;
+
+/**
+ * JdbcRecordWriter is wrapper class to write data to the underlying database.
+ */
+public class JdbcRecordWriter implements RecordWriter {
```

Comment:
> We could make `JdbcRecordWriter` extend `DBRecordWriter`? If there are any changes in this class behavior wrt `DBRecordWriter`, let's add comments to document that.

we cannot extend DBRecordWriter since it is a non-static inner class of DBOutputFormat